### DISCUSSION TITLE
Compile firmware version 3.0.2

### DISCUSSION THREAD

**Discussion by skimans** [iNavFlight/inav#7693] [General]
Hi friends
i have installed Ubuntu for modify firmware FLYWOOF411 for my quad.
But i want modifiy firmware 3.0.2 and not last firmware 5.0.0.
When i compile i obtain INAV_5.0.0_FLYWOOF411 but i want obtain INAV_3.0.2_FLYWOOF411.
How i can resolve ?
thanks

**Comment by MrD-RC** [comment]
You will need to rebase your branch to 3.0.2, if it exists.

**Reply by skimans** [reply]
Thanks MrD_RC i solved removing c:\Inav directory and put version 3.0.2 always in C:\inav.

**Comment by DzikuVx** [comment]
`git checkout 3.0.2`

**Reply by skimans** [reply]
> `git checkout 3.0.2`

but where i put git checkout 3.0.2 ??
after install clone of Inav ?
